packages/realm/src/Realm.ts

```diff
@@ -531,6 +700,15 @@ export class Realm {
     });
   }
 
+  /**
+   * Add a listener `callback` for the specified event `name`.
+   * @param eventName - The name of event that should cause the callback to be called.
+   *   DISCUSS: why does it say _Currently, only the "change" and "schema" events are supported_.
```

Comment:
current docs say only change and schema are supported. Docs website also doesn't mention beforenotify still. Is this intentional or just outdated?

Reply:
so I learnt now that beforeNotify was originally exposed in JS for primarily internal use, which is probably why the docs are like this. Perhaps it's worth not to mention beforeNotify at all not to confuse people about this niche event